salt/utils/thin.py

```diff
@@ -24,7 +24,7 @@
 
 import salt
 import salt.exceptions
-import salt.ext.tornado as tornado
+import tornado as tornado
```

Comment:
TODO (waiting for review to be done before making another commit)

```suggestion
import tornado
```